packages/webdriverio/src/commands/element/isEqual.js

```diff
@@ -0,0 +1,64 @@
+
+/**
+ *
+ * Return true if the selected DOM-element matches with the provided one.
+ *
+ * <example>
+    :isEqual.js
+    it('should detect if an element is clickable', () => {
+        const el = $('#el')
+        const sameEl = $('#el')
+        const anotherEl = $('#anotherEl')
+
+        el.isEqual(sameEl) // outputs: true
+
+        el.isEqual(anotherEl) // outputs: false
+    });
+ * </example>
+ *
+ * @alias element.isEqual
+ * @param   {Element}   el element to compare with
+ * @return  {Boolean}   true if elements are equal
+ *
+ */
+
+import { ELEMENT_KEY } from '../../constants'
+import { getBrowserObject } from '../../utils'
+
+const getWebElement = (el) => ({
+    [ELEMENT_KEY]: el.elementId, // w3c compatible
+    ELEMENT: el.elementId // jsonwp compatible
+})
+
+export default async function isEqual (el) {
+    const browser = getBrowserObject(this)
+
+    // mobile native
+    if (browser.isMobile) {
+        const context = await browser.getContext()
+        if (context.toLowerCase().includes('native')) {
+            const currentId = this.elementId
+            // just call any command that would refetch element if needed
+            try {
+                await this.getTagName()
```

Comment:
@christian-bromann do you think it's a correct way to do refetch?
I'm afraid that the element was found not with just `$(selector)` but with some custom strategy  or stuff like that

Reply:
ignore it, I don't think we need to refetch anything at this point